.github/scripts/test-info.mjs

```diff
@@ -573,6 +573,21 @@ const components = {
         requiredSecrets: ['AzureSqlServerConnectionString'],
         sourcePkg: ['state/sqlserver', 'internal/component/sql'],
     },
+    'state.gcp.firestore.docker': {
+        conformance: true,
+        requireDocker: true,
+        conformanceSetup: 'docker-compose.sh gcpfirestore',
+    },
+    'state.gcp.firestore.cloud': {
+        conformance: true,
+        requireGCPCredentials: true,
+        conformanceSetup: 'conformance-state.gcp.firestore-setup.sh',
```

Comment:
Do we need destroy scripts for both certification and conformance?

Reply:
`GPC Firestore`setup with **datastore mode** is really awkward. We have to manually configure it through the Google Cloud console.